configure.cmd

```diff
@@ -198,7 +215,7 @@ if "%BUILD_ENVIRONMENT%" == "MinGW" (
 ) else if %USE_CLANG_CL% == 1 (
     cmake -G %CMAKE_GENERATOR% -DCMAKE_TOOLCHAIN_FILE:FILEPATH=toolchain-msvc.cmake -DARCH:STRING=%ARCH% %BUILD_TOOLS_FLAG% -DUSE_CLANG_CL:BOOL=1 %* "%REACTOS_SOURCE_DIR%"
```

Comment:
The other cmake invocations probably should probably also be updated?

Reply:
Fixed